consul_exporter.go

```diff
@@ -173,38 +187,66 @@ func (e *Exporter) queryClient(services chan<- []*consul_api.ServiceEntry) {
 
 		services <- s_entries
 	}
+
+	c_entries, _, err := e.client.Health().State("any", &consul_api.QueryOptions{})
```

Comment:
camelCase instead of under_scores in Go variable names please. (dito the the existing `s_entries` above - somehow that slipped through).